src/components/frame/v5/pages/UserPreferencesPage/UserPreferencesPage.tsx

```diff
@@ -47,156 +43,149 @@ const UserPreferencesPage: FC<UserPreferencesPageProps> = ({
+              text={{
+                id: isCopied ? 'copy.addressCopied' : 'copy.address',
+              }}
+            />
           </div>
-        </form>
-      </TwoColumns>
-    </Spinner>
+        </div>
+        <span className="divider" />
+        <h5 className="heading-5">
+          {formatMessage({
+            id: 'userPreferencesPage.notificationPreferences',
+          })}
+        </h5>
+        <div className={styles.row}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.discord' }}
+            fieldDescription={{ id: 'description.discord' }}
+          />
+          <Button
+            mode="tertiary"
+            iconName="discord-logo"
+            iconSize="small"
+            text={{ id: 'button.connectDiscord' }}
+          />
+        </div>
+        <div className={styles.switchRow}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.browserNotification' }}
+            fieldDescription={{ id: 'description.browserNotification' }}
+          />
+          <Switch id="browser-notification" />
+        </div>
+        <div className={styles.switchRow}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.emailNotification' }}
+            fieldDescription={{ id: 'description.emailNotification' }}
+          />
+          <Switch id="email-notification" />
+        </div>
+        <span className="divider" />
+        <h5 className="heading-5">
+          {formatMessage({
+            id: 'userPreferencesPage.notificationFor',
+          })}
+        </h5>
+        <div className={styles.switchRow}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.mentionsNotification' }}
+            fieldDescription={{ id: 'description.mentionsNotification' }}
+          />
+          <Switch id="mentions" />
+        </div>
+        <div className={styles.switchRow}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.disableNotification' }}
+            fieldDescription={{ id: 'description.disableNotification' }}
+          />
+          <Switch id="disable-notifications" />
+        </div>
+        <span className="divider" />
+        <h5 className="heading-5">
+          {formatMessage({
+            id: 'userPreferencesPage.displayPreferences',
+          })}
+        </h5>
+        <div className={styles.switchRow}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.display' }}
+            fieldDescription={{ id: 'description.display' }}
+          />
+          <Switch id="display" />
+        </div>
+        <span className="divider" />
+        <div className={styles.row}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.switchAccount' }}
+            fieldDescription={{ id: 'description.switchAccount' }}
+          />
+          <Button
+            mode="tertiary"
+            iconName="user-switch"
+            iconSize="small"
+            text={{ id: 'button.switchAccount' }}
+          />
+        </div>
+      </div>
+    </form>
```

Comment:
There is a lot of repetition in this file, but then again, it was done before you came and updated it and it's not blocking us so I guess the best course of action would be to just take note of it and move on?

Reply:
Just copied it. But will change.